src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Safeguarding.cshtml

```diff
@@ -0,0 +1,68 @@
+﻿@page
+@model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.SafeguardingModel
+@{
+}
+
+<style>
+    a.nhs:link {
+        color: white;
+        background-color: transparent;
+        text-decoration: none;
+    }
+
+    a.nhs:visited {
+        color: white;
+        background-color: transparent;
+        text-decoration: none;
+    }
+
+    a.nhs:hover {
+        color: white;
+        background-color: transparent;
+        text-decoration: underline;
+    }
+
+    a.nhs:active {
+        color: black;
+        background-color: transparent;
+        text-decoration: underline;
+    }
+
+    .interrupt-panel {
+        width: 100%;
+        padding: 30px;
+        background: #005ea5;
+        color: #fff !important;
+        overflow: hidden;
+        width: auto;
+    }
+
+        .interrupt-panel h1, .interrupt-panel p, .interrupt-panel h2, .interrupt-panel ul {
+            color: #fff !important;
+        }
+
+</style>
+
+<a asp-page="/ProfessionalReferral/LocalOfferDetail" asp-route-id="@Guid.NewGuid().ToString()" asp-route-serviceId="@Model.Id" asp-route-name="@Model.Name" class="govuk-back-link">Back</a>
+
+<main class="govuk-main-wrapper custom-main" id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div class="interrupt-panel">
+                <h1 class="govuk-heading-l">
+                    Do not use this service to report safeguarding concerns
+                </h1>
+                <p class="govuk-body">Use the <a href="https://nhssafeguarding.app/?nocache=0.23461449202295054" id="nhs-app" class="nhs">NHS safeguarding app</a> for guidance on reporting safeguarding concerns.</p>
```

Comment:
We set [`$govuk-global-styles: true;`](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-global-styles), so we don't need to add `govuk-body` classes (or `govuk-link`), although it does no harm.

Reply:
Removed unwanted class